src/hint_processor/builtin_hint_processor/dict_manager.rs

```diff
@@ -323,7 +313,7 @@ mod tests {
                 Some(initial_dict)
             ))
         );
-        assert_eq!(segments.num_segments, 1);
+        assert_eq!(vm_proxy.segments.num_segments, 1);
```

Comment:
Will this fail when we turn vm_proxy.segments into a private field?

Reply:
resolved: https://github.com/lambdaclass/cairo-rs/pull/457/commits/441652db416c59a732477ba43525048061c6a791

Reply:
It is a bad idea to add a VMProxy method that is only used in testing.
Wont be better to assert `assert_eq!(vm.segments.num_segments, 1);`?

Reply:
Actually resolved(?): https://github.com/lambdaclass/cairo-rs/pull/457/commits/85b1591b4965b500a41972722a4a7f9af85a34c9